docs-partials/builder/vsphere/common/HardwareConfig-not-required.mdx

```diff
@@ -25,7 +25,10 @@
 - `displays` (int32) - Number of video displays. See [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-789C3913-1053-4850-A0F0-E29C3D32B6DA.html)
   for supported maximums. Defaults to 1.
 
-- `vgpu_profile` (string) - vGPU profile for accelerated graphics. See [NVIDIA GRID vGPU documentation](https://docs.nvidia.com/grid/latest/grid-vgpu-user-guide/index.html#configure-vmware-vsphere-vm-with-vgpu)
+- `pci_passthrough_allowed_device` ([]PCIPassthroughAllowedDevice) - Configure  Dynamic DirectPath I/O PCI Passthrough for virtual machine. See [vSphere documentation](https://blogs.vmware.com/vsphere/2020/03/vsphere-7-assignable-hardware.html)
```

Comment:
This is not official documentation. Please point to the product documentation on https://docs.vmware.com.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

Reply:
Also, there is an extra space - "Configure  Dynamic DirectPath"

Reply:
Thanks for noticing the space @tenthirtyam , added proper docs for vSphere docs and fixed space.